docs/learn/court.md

```diff
@@ -6,8 +6,298 @@ title: Decentralized Court
 Zeitgeist implements a decentralized court to handle disputes that may arise in
 the resolution of prediction markets outcomes.
 
-The court is inspired by [Aragon Court][] and [Kleros][], two projects that have
-already experimented with decentralized court systems.
+### General Introduction
 
-[aragon court]: https://example.com
-[kleros]: https://example.com
+The court system is responsible for ensuring that accurate information is added
+to the blockchain. Prediction markets, which rely on truthful data, reward
+traders who base their decisions on accurate information. If someone provides
+false information, they will be punished, while those who share accurate
+information will be rewarded. In essence, the court is a stake-weighted
+plurality decision-making machine. Zeitgeist's court makes use of the so-called
+[_Schelling point_][]. This is achieved by voting in secret and revealing the
+raw vote information later. The outcome with the most votes wins (plurality).
+The court serves as a dispute resolution mechanism. The court is inspired by
+[Kleros][], a project that has already experimented with an on-chain court
+system.
+
+### Dispute Management within Court
+
+If the oracle submits the wrong outcome due to malicious behaviour, disputes
+come into play in such cases. Anyone can dispute the oracle report, and once a
+dispute is triggered, the court takes over. The court is comprised of jurors and
+delegators who need to lock a certain amount of ZTG tokens to join the court
+system. The more tokens locked, the higher the probability of being selected as
+an active voter (juror) or delegator, who risks funds on behalf of delegated
```

Comment:
The term _voter_ is only mentioned here once. You seem to be using the term _active juror_ for the rest of the document.

Reply:
https://github.com/zeitgeistpm/documentation/pull/81/commits/a19875040e67869b6cc091dad09edd5163eef651